package.json

```diff
@@ -112,7 +112,7 @@
     ]
   },
   "volta": {
-    "node": "12.11.0",
+    "node": "16.16.0",
```

Comment:
Should we bump the engines property here as well?

Reply:
This should probably match the engines property but I changed it to match what I had. Seems like this would have been failing anyone previously using volta.

Reply:
Up to you guys on what this value should be. I can make the change

Reply:
yeah I use (and think mostly our team does) nvm and went to verify there wasn't a .nvmrc as well for the same reason.  My local env currently defaults to v14.